Model/Sync/Customers/Data.php

```diff
@@ -211,7 +212,7 @@ public function getCustomAttributeValue($customerId)
                 return $customAttributeValue;
             }
         }
-        return $customAttributeValue;
+        return $customAttributeValue;*/
```

Comment:
Why do we need this close comment symbol here?
Why not removing the commented code? (clean code)

Reply:
Hi Daniel,

We didn't remove this code because we might need this code in future if we decide to bring back the marketin checkbox on sign up page. But please let me know if we can remove this under the assumption that we won't be needed this in future.

Thanks,
Manaf

Reply:
Let's remove this commented code block and stick the clean code principal. If later we'll need this code back - we can add it back then.

Reply:
This is done. Please check